adventurous-anteaters/adv_ant/retroNet/static/retroNet/script.js

```diff
@@ -0,0 +1,51 @@
+// Functions for Interactive File Menu Bar 
+// - Click Butoon to Open Dropdown
+// - Clicking one dropdown closes all other
+// - Clicking outside the file menu bar will close all the dropdown.
+
+function open_dropdown(element_id) {
+  console.log('Opening Dropdown:', element_id)
```

Comment:
These should either have been `console.debug` calls, which I think is okay to leave in, or just removed.